packages/@okta/vuepress-theme-prose/components/SidebarItem.vue

```diff
@@ -9,7 +12,7 @@
           <a
             :href="href"
             @click="navigate"
-            :class="route.path === $route.path ? 'router-link-active' : ''"
+            :class="route.path === $route.path ? 'router-link-active link' : 'link'"
```

Comment:
Recommend we split this into the Vue class object binding to keep classnames separate:
```
{
  "router-link-active": route.path === $route.path,
  link: true,
}
```